packages/roosterjs-editor-plugins/lib/plugins/TableCellSelection/TableCellSelection.ts

```diff
@@ -91,6 +92,9 @@ export default class TableCellSelection implements EditorPlugin {
     onPluginEvent(event: PluginEvent) {
         if (this.editor) {
             switch (event.eventType) {
+                case PluginEventType.EditorReady:
+                    this.editor.queryElements('table', table => this.preventDuplicatedId(table));
+                    break;
```

Comment:
do we still need this change?

Reply:
I think you are right; we can just remove the duplicated id in selectTable, removing these changes here in the plugin